ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -234,3 +234,36 @@ def test_quantile(
         interpolation=interpolation[0],
         keepdims=keep_dims,
     )
+
+
+# nanquantile
+
+
+@handle_test(
+    fn_tree="functional.experimental.nanquantile",
+    dtype_x_axis=statistical_dtype_values(function="quantile"),
```

Comment:
You have to pass in the `safety factor` parameter here which will deal with the assertions, as the output values being generated are pretty close and I think for these statistical functions we are testing too extensively.
You can find the safety factor here, https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py#L16
It has the implementation in the function, you just have to pass the number, try not to pass a big value as it will restrict the dataset too much.